README.md

```diff
@@ -76,7 +76,10 @@ nginx -t -c /etc/nginx/nginx.conf
 For production, you would deploy this docker image to whatever servers you run your production services on.
 
 ### Option 3: Add locations to your existing Nginx config
-   If you already have servers running nginx, you can copy and paste the locations from the [nginx.conf](https://github.com/mixpanel/tracking-proxy/blob/master/nginx.conf) file in this repo and adjust the locations to match your preference.
+If you already have servers running nginx, you can copy and paste the locations from the [nginx.conf](https://github.com/mixpanel/tracking-proxy/blob/master/nginx.conf) file in this repo and adjust the locations to match your preference.
+
+### Additional Configuration
+In case you see events tracked to your project, but you the geo-location is not being added, there's likely a configuration issue with passing the IP address through the headers. Within the [nginx configuration]([url](https://github.com/mixpanel/tracking-proxy/blob/2c08e999d4b38aa943fad55884bcfe0ef72bb681/nginx.conf#L31)), by default, we leverage `$http_x_forwarded_for` and pass it through the `X-Real-IP` header to forward the IP from the original client; in some providers, this is not avaialble and you will want to review which one provides the IP address from the client. As an example in Cloudflare, you need to pass `$realip_remote_addr`instead. You'll want to review the option that applies to your cloud provider.
```

Comment:
Corrected a few typos here, but a slightly more substantive change around the phrasing "geo-location is not being added": in the case where geoip props are not present at all, usually that means the events were tracked without the `ip=1` param, which is separate from proxy config; whereas incorrect client-ip forwarding would typically result in incorrect geoip data (based on the IP of the proxy server as seen by mixpanel's API servers).
```suggestion
In case you see events tracked to your project, but geo-location properties are not being added correctly, there's likely a configuration issue with passing the IP address through the headers. Within the [nginx configuration]([url](https://github.com/mixpanel/tracking-proxy/blob/2c08e999d4b38aa943fad55884bcfe0ef72bb681/nginx.conf#L31)), by default, we leverage `$http_x_forwarded_for` and pass it through the `X-Real-IP` header to forward the IP from the original client; in some providers, this is not available and you will want to review which header/variable provides the IP address from the client. As an example, in Cloudflare, you need to pass `$realip_remote_addr` instead. You'll want to review the option that applies to your cloud provider.
```